addons/l10n_in/data/account.account.tag.csv

```diff
@@ -45,3 +45,5 @@
 "tax_tag_cess_rc","CESS (RC)","taxes"
 "tax_tag_non_itc_cess_rc","NON ITC CESS (RC)","taxes"
 "tax_tag_other_non_itc_cess_rc","Other NON ITC CESS (RC)","taxes"
+"tax_tag_base_tds","BASE TDS","taxes"
+"tax_tag_base_tcs","BASE TCS","taxes"
```

Comment:
> We display a message when those tags are used. So these tags mean "low rates", whatever that means?

No, it doesn't mean low rate, it's just mean that TDS/TCS is applicable on this transaction at higher rate, as we don't have a field on taxes to define higher rate.

Reply:
Why don't you use the tags to say that there is something with higher rates then?

Reply:
> Why don't you use the tags to say that there is something with higher rates then? Or the xmlids in stable and a field in master? Or something else?
> 
> I just think that this warning is going to be confusing more than anything.

Thank you for the idea. I think it's a good one, so we're updating the plan. Now, instead of BASE TAG, we'll have two tags: "Higher TDS" and "Higher TCS". We'll use these tags when the TDS/TCS rates are high. If there's no tag and PAN isn't mentioned, we'll just give a warning. But if either of those two tags is present and PAN isn't there, the system won't show an alert.

Reply:
Doesn't it make more sense to know which are *not* the highest rates, so that you know exactly when to have a warning/error?

Reply:
> Doesn't it make more sense to know which are _not_ the highest rates, so that you know exactly when to have a warning/error?

Yes, it makes sense. That's why we labeled regular rates as "BASE TDS" and "BASE TCS" and higher rates as "HIGH TDS" and "HIGH TCS." 

Additionally, we're going to include these four labels in the new TDS report/spreadsheet we're making because the current one doesn't show the base amount.